### PR TITLE
Release v7.10.0

### PR DESCRIPTION
**Pull request by lildude**
Three whole COVID-19 filled months since the last release so time for another.

- [x] Test locally and in GitHub.com staging
- [ ] Deploy to GitHub.com
- [ ] Push new gem
- [ ] Deploy grammar update to syntax highlighter
- [ ] Update Lightshow
- [ ] 🎉 

_[ Template removed as it doesn't apply ]_

### CODE REVIEW

File: vendor/licenses/grammar/Atom-PostScript.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 type: grammar
 name: Atom-PostScript
-version: 7f562cdfd751ae1863d6b88621e82cd27511ba8a
+version: 5d33239f382c26e96daa1db7dd42559d824b2485
```
Comment (Alhadis) [inline vendor/licenses/grammar/Atom-PostScript.txt:4]: Could you do that again?

Nah, I'm kidding. ❤️  Thanks!

Comment (lildude) [reply vendor/licenses/grammar/Atom-PostScript.txt:4]: 🤔 that seems to have only updated the license and not the submodule ref.

Comment (lildude) [reply vendor/licenses/grammar/Atom-PostScript.txt:4]: Sorted!

**Comment by lildude** [comment]
Linguist v7.10.0 is now live on GitHub.com.

**Pull request merged**